code/__HELPERS/icons.dm

```diff
@@ -731,10 +731,16 @@ The _flatIcons list is a cache for generated icon files.
 
 	var/icon/add // Icon of overlay being added
 
-		// Current dimensions of flattened icon
-	var/{flatX1=1;flatX2=flat.Width();flatY1=1;flatY2=flat.Height()}

```

Comment:
One of the two places in the code where this format is used. It's ugly and harder to understand; it really shouldn't be used.

Reply:
_Why_ would anyone do that?